scripts/obtain/dotnet-install.sh

```diff
@@ -656,7 +656,9 @@ install_dotnet() {
     say_verbose "Zip path: $zip_path"
 
     say "Downloading link: $download_link"
-    download "$download_link" $zip_path || download_failed=true
+    # Failures are expected in legacy case. Do not output to stderr, since
+    # the output stderr are considered an error
+    download "$download_link" $zip_path 2>&1 || download_failed=true
```

Comment:
Failures are expected in the _non-_legacy case

Reply:
Fixed